lib/driverProviders/sauce.js

```diff
@@ -56,7 +56,8 @@ SauceDriverProvider.prototype.setupEnv = function() {
   var deferred = q.defer();
   this.sauceServer_ = new SauceLabs({
     username: this.config_.sauceUser,
-    password: this.config_.sauceKey
+    password: this.config_.sauceKey,
+    agent: this.config_.agent
```

Comment:
Can you switch this variable name to `sauceAgent`? Many settings are shared in the config so just having something called 'agent' could be confusing.

Also, please add documentation in `docs/referenceConf.js` for the new field.